adaptive_scheduler/interfaces.py

```diff
@@ -6,7 +6,7 @@
 from datetime import datetime, timedelta
 from redis import Redis
 
-redis = Redis(host=os.getenv('REDIS_URL', 'redisdev'), db=0, socket_connect_timeout=15,
+redis = Redis.from_url(url=os.getenv('REDIS_URL', 'redis://redisdev'), socket_connect_timeout=15,
```

Comment:
Should make the defaults the same between the two I uses I think - I realize it wasn't the same before but probably should have been